### ISSUE TITLE
加载预训练模型出错

### ISSUE THREAD

**Issue by summerrr** [DetectionTeamUCAS/Faster-RCNN_Tensorflow#13]
2018-09-19 19:58:32.524151: W tensorflow/core/util/tensor_slice_reader.cc:95] Could not open /home/user10/notebook/Faster-RCNN_Tensorflow/data/pretrained_weights/resnet_v1_101.ckpt: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?加载与训练模型的时候会出错呢，好像是不匹配

**Comment by summerrr** [comment]
还有是在哪里选择预训练模型的呢，我改了半天想用resnet50，结果一直是101层的

**Comment by yangxue0827** [comment]
[here](https://github.com/DetectionTeamUCAS/Faster-RCNN_Tensorflow/blob/master/libs/configs/cfgs.py#L54)
resnet_v1_101改成resnet_v1_50

**Comment by yangxue0827** [comment]
[here](https://github.com/DetectionTeamUCAS/Faster-RCNN_Tensorflow/blob/master/libs/configs/cfgs.py#L54)
resnet_v1_101改成resnet_v1_50 @summerrr
